docs/using-smallrye-stork.md

```diff
@@ -0,0 +1,50 @@
+# Using SmallRye Stork
+
+SmallRye GraphQL client (both dynamic and typesafe) supports SmallRye Stork for 
+service discovery and load balancing. If your client is configured to connect to a URI
+with the `stork://` or `stork://` scheme, then Stork will be used to determine the
```

Comment:
They are the same (stork://)

Reply:
hmmmmmm https://github.com/smallrye/smallrye-graphql/pull/1344